components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -467,4 +467,24 @@ - (void)testMessageViewAdjustsFontForContentSizeCategoryWhenScaledFontIsUnavaila
   XCTAssertTrue(messageView.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable);
 }
 
+- (void)testMessageStaysWhenFocusOnShowIsEnabled {
+  // Given
+  self.manager.internalManager.isVoiceOverRunningOverride = YES;
+  self.message.focusOnShow = YES;
+  self.message.duration = 0.1;
+
+  // When
+  [self.manager showMessage:self.message];
+  XCTestExpectation *expectation = [self expectationWithDescription:@"completed"];
+  dispatch_time_t popTime =
+      dispatch_time(DISPATCH_TIME_NOW, (int64_t)((CGFloat)0.1 * NSEC_PER_SEC));
```

Comment:
This duration should probably be longer than `self.message.duration` to account for the animations in and out